### ISSUE TITLE
Broken on PARADISE LOST (and presumably older versions)

### ISSUE THREAD

**Issue by beer-psi** [beer-psi/saekawa#2]
```
Could not parse request body: missing field `classEmblemBase` at line 1 column 1258
```

Older versions of CHUNITHM don't have `classEmblemBase` and `classEmblemMedal`. Presumably the `medal` field in `userDataEx` replaces this?

**Comment by beer-psi** [comment]
Might be fixed by 0.4? At least the dan/emblem fields are optional now, but I'm not sure if userDataEx.medal contains dan information on old CHUNITHM.

**Comment by TehCupcakes** [comment]
There's still an issue with Paradise Lost. Ever since 0.4.0, trying to run with saekawa will simply crash the game on boot. This is my log:

```
2026-02-03 01:19:24 INFO  [saekawa] saekawa 0.4.7 (a28173b@trunk) starting up...
2026-02-03 01:19:24 INFO  [saekawa::config] Config file: saekawa.toml
2026-02-03 01:19:24 INFO  [saekawa::updater] Checking for updates...
2026-02-03 01:19:24 INFO  [saekawa::updater] Already up-to-date.
2026-02-03 01:19:24 INFO  [saekawa::saekawa] Loaded API keys for 2 access code(s).
2026-02-03 01:19:24 INFO  [saekawa::saekawa] Running on SDBT 1.50.00
2026-02-03 01:19:24 INFO  [saekawa::saekawa] Network requests are encrypted.
```
(Also tried all the other 0.4.x releases.)

There's no indication of what went wrong; after this it simply fails with the generic `Game processes have terminated` message.

I noticed the "Network requests are encrypted" message, which it seems to say regardless of whether I have the "No Encryption" and "No TLS" patches enabled or not.

I tried 0.2.2 and 0.3.4. The game launches fine with these, but it throws the error about `classEmblemBase` when it tries to upload:
```
ERROR saekawa::saekawa -> Could not parse request body: missing field `classEmblemBase` at line 1 column 1372
```

So long story short, there seems to have been a regression in 0.4.0, even if it did fix other issues that were present before.

**Comment by beer-psi** [comment]
I believe the automatic encryption detection is specific to post-NEW versions and it has also been broken recently in X-VERSE. Not really sure how to approach this issue. I might just bail and store encryption keys in the DLL.
